ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -431,9 +435,11 @@ tests:
       TEST_SUITE: full
     workflow: assisted-baremetal
 - as: e2e-metal-assisted-kube-api-net-suite
-  optional: true
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|PROJECT|LICENSE)$
   steps:
     cluster_profile: packet-assisted
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:stable-4-11
```

Comment:
why do we need stable 4.11 here? is there a bug-fix that exists in 4.11 but not in latest 4.12 (which should be the default afaik)

Reply:
Removed on your advice here.
At the time of writing, your patch had not yet been merged or it was too close to the merge time to be sure that it would be picked up properly.
That's no longer an issue now so I am removing this.